### PR TITLE
Gal 473/lsoa midpoint calc

### PR DESCRIPTION
**Pull request by Abdul-NHS**
## Description
Create new csv with average easting and average northing attached to the lsoa csv file, and remove postcodes

## Type of changes

- [ ] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: scripts/pipeline/nonprod_postcode_load/nonprod_postcode_load.py [hunk 1]
```diff
@@ -100,9 +100,10 @@ def batch_write_to_dynamodb(lsoa_data):
     # format and send these to the batch write function
     # repeat till no records are left
     dynamodb_client = boto3.client('dynamodb')
-    for i in range(1, 100000, 100):
-        upper_bound_slice = i+100
+    for i in range(1, 10, 1):
```
Comment (zainm97) [inline scripts/pipeline/nonprod_postcode_load/nonprod_postcode_load.py:103]: think this was a rebase issue so my fault. Can you revert this back to the previous change

File: scripts/pipeline/seed-data.sh [hunk 2]
```diff
@@ -10,22 +10,22 @@ set -e
 # ==============================================================================
 
 function main() {
-  echo Initiating upload of Participating ICBs test data to database
+  # echo Initiating upload of Participating ICBs test data to database
```
Comment (zainm97) [inline scripts/pipeline/seed-data.sh:13]: think this was a rebase issue so my fault. Can you revert this back to the previous change

File: scripts/pipeline/seed-data.sh [hunk 3]
```diff
@@ -51,7 +53,7 @@ function main() {
 
   mkdir nonprod-lsoa-data
 
-  aws s3 cp s3://galleri-ons-data/lsoa_data/unique_lsoa_data.csv ./nonprod-lsoa-data
+  aws s3 cp s3://galleri-ons-data/lsoa_data_/unique_lsoa_data.csv ./nonprod-lsoa-data
```
Comment (zainm97) [inline scripts/pipeline/seed-data.sh:56]: think this was a rebase issue so my fault. Can you revert this back to the previous change

File: terraform/environment/dev/resources/lambda/getLsoaInRange/lambdaHandler/getLsoaInRangeLambda.js [hunk 4]
```diff
@@ -9,9 +9,9 @@ const MTOKM = 1000;
 */
 export const handler = async (event, context) => {
   const start = Date.now();
-  // console.log(
-  //   "*************\n Event = " + JSON.stringify(event, null, 2) + "\n**********"
-  // );
+  console.log(
+    "*************\n Event = " + JSON.stringify(event, null, 2) + "\n**********"
```
Comment (zainm97) [inline terraform/environment/dev/resources/lambda/getLsoaInRange/lambdaHandler/getLsoaInRangeLambda.js:13]: think this was a rebase issue so my fault. Can you revert this back to the previous change

File: terraform/environment/dev/resources/lambda/getLsoaParticipants/lambdaHandler/getLsoaParticipantsLambda.js [hunk 5]
```diff
@@ -11,33 +11,35 @@ export const handler = async (event, context) => {
...
-  },
-  {
-    IMD_DECILE: { N: '9' },
-    NORTHING_1M: { N: '228441' },
-    EASTING_1M: { N: '509665' },
-    FORECAST_UPTAKE: { N: '1' },
-    LSOA_2011: { S: 'E01015555' },
-    DISTANCE_TO_SITE: { N: '14.19' }
-  }];
+  const lsoaList = [
```
Comment (zainm97) [inline terraform/environment/dev/resources/lambda/getLsoaParticipants/lambdaHandler/getLsoaParticipantsLambda.js:14]: think this was a rebase issue so my fault. Can you revert this file back to the previous change

File: scripts/pipeline/nonprod_postcode_load/nonprod_postcode_load.py (hunk 1 above)
Comment (Abdul-NHS) [reply scripts/pipeline/nonprod_postcode_load/nonprod_postcode_load.py]: Reverted change

File: scripts/pipeline/seed-data.sh (hunk 2 above)
Comment (Abdul-NHS) [reply scripts/pipeline/seed-data.sh]: Reverted change

File: terraform/environment/dev/resources/lambda/getLsoaInRange/lambdaHandler/getLsoaInRangeLambda.js (hunk 4 above)
Comment (Abdul-NHS) [reply terraform/environment/dev/resources/lambda/getLsoaInRange/lambdaHandler/getLsoaInRangeLambda.js]: reverted change

File: terraform/environment/dev/resources/lambda/getLsoaParticipants/lambdaHandler/getLsoaParticipantsLambda.js (hunk 5 above)
Comment (Abdul-NHS) [reply terraform/environment/dev/resources/lambda/getLsoaParticipants/lambdaHandler/getLsoaParticipantsLambda.js]: Have actioned by removing prettier formating

**Review by zainm97 — CHANGES_REQUESTED** [review]
All looks good, could you just move the uploadUniqueLsoa.js into the test_data folder and happy to approve

**Review by zainm97 — APPROVED** [review]
Good stuff. I'll adjust the z_rename folder in Gal-403

**Pull request closed without merging**